src/components/MapTitleBar.js

```diff
@@ -39,7 +39,7 @@ const MapTitleBar = () => {
         if (ref.current.textContent.trim() === '') {
             ref.current.textContent = UNTITLED_NAME;
         }
-        const name = ref.current.textContent;
+        const name = ref.current.textContent.slice(0, 30);
```

Comment:
I think if we're going to have a character limit, we should stop the user entering more, rather than truncating it afterwards, since it might not be obvious that we did that.

Also, 30 characters seems quite a low limit- is it possible to make it bigger or would this mess up the layout too much?
An option could be to make the name scrollable?

Reply:
@rogup discussed name limit with lynne here: https://github.com/DigitalCommons/land-explorer-front-end/issues/187#issuecomment-1520325441

basically going to keep it short until someone has a real need to have it long. 30 chars is the length of `longest possible map name` 

agree that should block entering, i'll make another commit with that in